Add documentation for json schema usage to validate config in the FAQ (#9340)
traefik/traefik: docs/content/getting-started/faq.md

For more details,
please check out the [forwarded header](../routing/entrypoints.md#forwarded-headers) documentation.

> ## What means the "field not found" error?

Reviewer: ```suggestion
## What does the "field not found" error mean?
```

```shell
error: field not found, node: -badField-
```

> The "field not found" error occurs when an unknown property is found, in the dynamic or static configuration.

Reviewer: ```suggestion
The "field not found" error occurs, when an unknown property is encountered in the dynamic or static configuration.
```

One easy way to check whether a configuration file is well-formed, is to validate it with:

- [JSON Schema of the static configuration](https://json.schemastore.org/traefik-v2.json)
- [JSON Schema of the dynamic configuration](https://json.schemastore.org/traefik-v2-file-provider.json)

> ## Why some resources (routers, middlewares, services...) are not created/applied?

Reviewer: ```suggestion
## Why are some resources (routers, middlewares, services...) not created/applied?
```

> As a common tip, if a resource is dropped/not created by Traefik after evaluating the dynamic configuration,

Reviewer: ```suggestion
As a common tip, if a resource is dropped/not created by Traefik after the dynamic configuration was evaluated,
```

one should look up for an error in the logs.

If found, the error obviously confirms that something went wrong while creating the resource,

> and the reason can help to correct a bad configuration.

Reviewer: ```suggestion
and the message should help in figuring out the mistake in the configuration, and how to fix it.
```
